sys/dev/bwn/if_bwn_pci.c

```diff
@@ -295,6 +295,10 @@ DEFINE_CLASS_0(bwn_pci, bwn_pci_driver, bwn_pci_methods,
     sizeof(struct bwn_pci_softc));
 DRIVER_MODULE_ORDERED(bwn_pci, pci, bwn_pci_driver, bwn_pci_devclass, NULL,
     NULL, SI_ORDER_ANY);
+MODULE_PNP_INFO("U16:vendor;U16:device;D:#", pci, bwn_pci,
```

Comment:
For the unique string, I'd be tempted to make this `bwn_siba` instead of `bwn_pci` to give a hint this is related to [siba(4)](https://www.freebsd.org/cgi/man.cgi?query=siba&apropos=0&sektion=0&manpath=FreeBSD+11.1-RELEASE+and+Ports&arch=default&format=html)

Reply:
Yep, changed it as bwn_siba